### ISSUE TITLE
TypeError after mpv exits.

### ISSUE THREAD

**Issue by mweinelt** [rejuvyesh/rmpv#1]
Running rmpc against some-video-mkv, then forwarding to the end and when mpv exits rmpv seems to be not working.

```
/home/hexa/.gem/ruby/2.2.0/gems/rmpv-0.1.7/bin/rmpv:73:in `kill': wrong argument type nil (expected VM/thread) (TypeError)
    from /home/hexa/.gem/ruby/2.2.0/gems/rmpv-0.1.7/bin/rmpv:73:in `<top (required)>'
    from /home/hexa/.gem/ruby/2.2.0/bin/rmpv:23:in `load'
    from /home/hexa/.gem/ruby/2.2.0/bin/rmpv:23:in `<main>'
```

I set up only a trakt account in ~/.rmpvrc, that should be sufficient, right? Also does this work against the new site already? Not really sure when that changed. If not, are you planning to update it or is this project dead?

**Comment by rejuvyesh** [comment]
I am really sorry. Haven't gotten around to updating it for the new api. Pull requests are of course welcome. Planning to update it this summer.

**Comment by dgw** [comment]
I see that the traktr gem hasn't been updated since Q1 2014. I assume that's what handles the actual API interaction, and therefore that's why rmpv hasn't been updated for Trakt's v2 API.

**Comment by rejuvyesh** [comment]
You are right. This is mostly abandoned right now :cry: . Let me know if you want to take over. If I were to do it now I'll probably use `mpv`'s own `lua` scripting capabilities.
